mrp_production_request/wizards/mrp_production_request_create_mo.py

```diff
@@ -121,6 +122,11 @@ def create_mo(self):
         self.ensure_one()
         vals = self._prepare_manufacturing_order()
         mo = self.env["mrp.production"].create(vals)
+        move = mo._get_moves_raw_values()
+        component_ids = []
+        for rec in move:
+            component_ids.append((0, 0, rec))
+        mo.write({"move_raw_ids": component_ids})
```

Comment:
I think you only have to confirm it, I'm v13 they added back the state "draft".

Reply:
I just wanted to do same as v12 flow. Yes, In v13, Draft state is new added in MRP Production.

Reply:
Yes, I think this is fine. I am checking and from in draft state you already have raw moves. So your fix is correct.

Can you squash this fix with the previous commit?

Reply:
Sure.

Reply:
> Yes, I think this is fine. I am checking and from in draft state you already have raw moves. So your fix is correct.
> 
> Can you squash this fix with the previous commit?

Squash done !